docs-src/concepts/what-are-actvitiy-events.md

```diff
@@ -0,0 +1,28 @@
+---
+id: what-are-activity-events
+title: What are Activity Events?
+sidebar_label: Activity Events
+description: There are seven Activity Events that are added to History at different points in an Activity Execution.
+tags:
+  - term
+  - explanation
+---
+
+There are seven Activity-related [Events](/workflows#event) that are added to History at different points in an Activity Execution.
+
+There are two important things to note when matching Activity Execution lifecycle to Activity Events:
+
+- In Event names, "ActivityTask" refers to an [Activity Execution](/activities#activity-execution), not an [Activity Task](/tasks#activity-task).
+- While the Activity is running and retrying, [ActivityTaskScheduled](/references/events#activitytaskscheduled) is the only Activity-related event in History: [ActivityTaskStarted](/references/events#activitytaskstarted) is written along with a terminal event like [ActivityTaskCompleted](/references/events#activitytaskcompleted) or [ActivityTaskFailed](/references/events#activitytaskfailed).
```

Comment:
David noted that the exception to this rule is that if there is no Retry Policy attached to an Activity Execution, ActivityTaskStarted is written immediately. @bergundy is it possible to start an Activity without a Retry Policy? I thought it always gets the default Retry Policy if not specified

Reply:
I don't know under which conditions `HasRetryPolicy` would be false.
I think we always let the server fill in the default retry policy.
@dnr does does `HasRetryPolicy` imply the activity cannot be retried (e.g. max attempts 1)?

Reply:
As far as I can tell, this is never false in the current server code. It may have been possible in the past. So no need to worry about it (and I guess we could clean up some code if we want)